.github/workflows/sync-main-to-stable.yaml

```diff
@@ -30,12 +30,12 @@ jobs:
         uses: actions/checkout@93cb6efe18208431cddfb8368fd83d5badbf9bfd # v5.0.1
         with:
           fetch-depth: 0
-          ref: rhoai
+          ref: stable
           token: ${{ steps.generate-token.outputs.token }}
 
-      - name: Update main commits into rhoai
-        id: update_main_into_rhoai
+      - name: Update main commits into stable
+        id: update_main_into_stable
         run: |
-          git fetch origin main rhoai
-          git merge origin/main
-          git push origin rhoai
+          git fetch origin main stable
```

Comment:
stable is already fetched by actions/checkout, right?

From the [docs](https://github.com/actions/checkout?tab=readme-ov-file#checkout-v4)

```
Only a single commit is fetched by default, for the ref/SHA that triggered the workflow.
```